spec/features/post_a_peep_spec.rb

```diff
@@ -3,7 +3,9 @@
         visit('/peeps/new')
         fill_in('name', with: 'Test peep')
         
+        
         click_button('Post')
+        
 
         expect(page).to have_content 'Test peep'
```

Comment:
maybe you could add another line here something like expect(page).to have_content 'posted at 19.50' or something like that to test your timestamp?